tests/components/roborock/test_init.py

```diff
@@ -38,3 +40,15 @@ async def test_config_entry_not_ready(
     ):
         await async_setup_component(hass, DOMAIN, {})
         assert mock_roborock_entry.state is ConfigEntryState.SETUP_RETRY
+
+
+async def test_reauth_started(
+    hass: HomeAssistant, bypass_api_fixture, mock_roborock_entry: MockConfigEntry
+) -> None:
+    """Test reauth flow started."""
+    with patch(
+        "homeassistant.components.roborock.RoborockApiClient.get_home_data",
```

Comment:
Consider also testing this case with `RoborockException` given the test coverage message

Reply:
Added the test from my other PR for simplicity sake